src/game/ingame_menu.c

```diff
@@ -1054,41 +1038,42 @@ void render_dialog_triangle_next(s8 linesPerBox) {
     gSPPopMatrix(gDisplayListHead++, G_MTX_MODELVIEW);
 }
 
+// King Bob-omb (Start), Whomp (Start), King Bob-omb (throw him out), Eyerock (Start), Wiggler (Start)
+static s16 sDialogBossStart[] = { DIALOG_017, DIALOG_114, DIALOG_128, DIALOG_117, DIALOG_150 };
+// Koopa the Quick (BoB), Koopa the Quick (THI), Penguin Race, Fat Penguin Race (120 stars)
+static s16 sDialogRaceSound[] = { DIALOG_005, DIALOG_009, DIALOG_055, DIALOG_164             };
+// Red Switch, Green Switch, Blue Switch, 100 coins star, Bowser Red Coin Star
+static s16 sDialogStarSound[] = { DIALOG_010, DIALOG_011, DIALOG_012, DIALOG_013, DIALOG_014 };
+// King Bob-omb (Start), Whomp (Defeated), King Bob-omb (Defeated, missing in JP), Eyerock (Defeated), Wiggler (Defeated)
+static s16 sDialogBossStop[]  = { DIALOG_017, DIALOG_115, DIALOG_116, DIALOG_118, DIALOG_152 };
+
 void handle_special_dialog_text(s16 dialogID) { // dialog ID tables, in order
```

Comment:
i mean sure but maybe out of scope. only reason i moved these is because they werent static and i'm pretty sure gcc was copying these arrays into the stack every single time the function was called.